packages/@uppy/companion/src/server/s3-client.js

```diff
@@ -29,6 +29,7 @@ module.exports = (companionOptions, createPresignedPostMode = false) => {
     /** @type {import('@aws-sdk/client-s3').S3ClientConfig} */
     let s3ClientOptions = {
       region: s3.region,
+      forcePathStyle: Boolean(s3.forcePathStyle)
```

Comment:
why do we need to convert it to a boolean?
```suggestion
      forcePathStyle: s3.forcePathStyle
```